examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/ipex/README.md

```diff
@@ -0,0 +1,29 @@
+Step-by-Step
+============
+This document describes the step-by-step instructions to run llama2 SmoothQuant with Intel® Neural Compressor and Intel® Extension for PyTorch.
+
+# Prerequisite
+```
+# Installation dependencies
+pip install -r requirements.txt
+```
+
+# Run Quantization
+
+## Llama-2-7b
+```bash
+python run_llama2_sq.py \
+    --model-id meta-llama/Llama-2-7b-hf \
+    --batch-size 56 \
+    --sq-recipes "llama2-7b"
+```
+## Llama-2-13b
+```bash
+python run_llama2_sq.py \
+    --model-id meta-llama/Llama-2-13b-hf \
+    --batch-size 56 \
+    --sq-recipes "llama2-13b"
+```
+> Notes:  
+> - INT8 model will be saved into "./saved_results", and the output model can be loaded and evaluated by IPEX.  
```

Comment:
How about listing generated files in saved_results. In some case,  user may only take qconfig file which reflects the tuning result.  for example, in IPEX LLM example (https://github.com/intel-innersource/frameworks.ai.pytorch.ipex-cpu/tree/release/2.1/examples/cpu/inference/python/llm#static-quantization-int8), it does quantization itself but take our tuning result config as input

Reply:
OK, I will list the files in saved_results